users/views.py

```diff
@@ -51,15 +53,44 @@ def UserCreateConfirmView(request, token):
         user.is_active = True
         user.save()
 
-        return redirect('users:signup')
+        return redirect('users:login')
     except ObjectDoesNotExist:
         return redirect('users:signup')
 
 
+def UserLoginView(request):
+    if request.method == "GET":
+        template = loader.get_template('users/user_login.html')
+        return HttpResponse(template.render(request=request))
+
+    if request.method == "POST":
+        email = request.POST['email']
+        password = request.POST['password']
+
+        user = authenticate(email=email, password=password)
+
+        if user is not None:
+            if user.is_active:
+                login(request, user)
+                return redirect('users:profile', pk=user.pk)
+            else:
+                messages.error(request, 'Your account is inactive. Be sure to check your emails.')
+                return redirect('users:login')
+        else:
+            messages.error(request, message='Bad credentials.')
+            return redirect('users:login')
+
+
+def UserLogoutView(request):
+    logout(request)
+    return redirect('landingpage')
```

Comment:
Should we really always redirect to the landing page instead of just refreshing the current page when possible?
E.g. you view your profile, click `Logout`, then you still see your profile, but can't edit it for example.